sample_project/Source/sample_project/Measure/Measure.cpp

```diff
@@ -117,11 +117,23 @@ void AMeasure::AddStop()
 			GeodeticDistanceText = FString::Printf(TEXT("Distance: %f %s"), round(GeodeticDistance * 1000.0) / 1000.0, *UnitText);
 			UIWidget->ProcessEvent(WidgetFunction, &GeodeticDistanceText);
 
-			FeaturePoints.Add(lastStop);
+			// Confirm FeaturePoints list does not already contain element
+			if (!FeaturePoints.Contains(lastStop))
+			{
+				FeaturePoints.Add(lastStop);
+			}
 			Interpolate(lastStop, lineMarker);
-			FeaturePoints.Add(lineMarker);
+			// Confirm FeaturePoints list does not already contain element
```

Comment:
Make sure there's a space above `if` statments and their comments
```suggestion
			
			// Confirm FeaturePoints list does not already contain element
```